developers/weaviate/configuration/rbac/manage-users.mdx

```diff
@@ -0,0 +1,476 @@
+---
+title: Manage users
+sidebar_label: Manage users
+sidebar_position: 1
+image: og/docs/configuration.jpg
+# tags: ['rbac', 'roles', 'configuration', 'authorization']
+---
+
+import Link from '@docusaurus/Link';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.permissions.py';
+import UserPyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.users.py';
+import OidcUserPyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.oidc.users.py';
+
+:::info Added in `v1.29` and `v1.30`
+Role-based access control (RBAC) is generally available in Weaviate from version `v1.29`.  
+User management is available from version `v1.30`.
+:::
+
+In Weaviate, Role-based access control (RBAC) allows you to define roles and assign permissions to those roles. Users can then be assigned to roles and inherit the permissions associated with those roles.
+
+On this page, you will find examples of how to programmatically **manage users** and their associated roles with Weaviate client libraries.
```

Comment:
Hey @g-despot I think this was ported over from an earlier version, but maybe it could be reviewed once again. 

Reading this page, I wonder if "User" here could be clarified to use the term "Database user" where applicable as Sebastian mentioned in our discussion, to contrast against "OIDC user". 

Then, user could mean: Union[Database user, OIDC user]. Wdyt?

Reply:
I agree, this could be a good place to introduce the different user types. I will also try to follow this naming convention where it makes sense `Union[Database user, OIDC user]`